calculator.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+
+int main(){
+	float x;
+	float y;
+	float ans;
+	char op;
+	printf("Enter first number:\n");
+	scanf("%f", &x);
+	printf("Enter second number:\n");
+	scanf("%f", &y);
+	printf("Would you like to +, -, *, /, or %%?\n");
+	scanf(" %c", &op);
+
+	switch(op)
+	{
+		case ('+'):
+			ans = x + y;
+			printf("Your answer is ");
```

Comment:
you could have combined these print statements - not a big deal, but just thought I'd let you know.